sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -21,6 +21,9 @@ class AdminOperations {
                 let user = User(propertyDictionary: userDict as! [String: AnyObject])
                 userArray.append(user)
             }
+            userArray.sortInPlace({ (user1, user2) -> Bool in
```

Comment:
make this a class function for User since it seems to be used a lot. then we can just pass in that function object as the comparator